server/public/model/builtin.go

```diff
@@ -3,11 +3,14 @@
 
 package model
 
+// These functions are deprecated. Please use only NewPointer.
 func NewBool(b bool) *bool       { return &b }
 func NewInt(n int) *int          { return &n }
 func NewInt64(n int64) *int64    { return &n }
 func NewString(s string) *string { return &s }
-func NewPointer[T any](t T) *T   { return &t }
```

Comment:
When will we remove these?

Reply:
Probably with the next ESR. Technically these could go away right now, but just wanted to give a bit of leeway to other consumers.